app/protocol/flows/blockrelay/ibd.go

```diff
@@ -55,47 +59,55 @@ func (flow *handleRelayInvsFlow) runIBDIfNotRunning(highHash *externalapi.Domain
 	return nil
 }
 
-func (flow *handleRelayInvsFlow) syncHeaders(highHash *externalapi.DomainHash) error {
+func (flow *handleRelayInvsFlow) syncHeaders(highHash *externalapi.DomainHash) (bool, error) {
```

Comment:
It could be nice to have a comment on syncHeaders explaining what does the bool return means and when can it be false